### PR TITLE
Add a new CLI config "existing-output-dir" that

### PR DESCRIPTION
**Pull request by travisbloom**
allows users to pass in a previously built storybook
directory.

### CODE REVIEW

File: bin/storybook_to_ghpages [hunk 1]
```diff
@@ -6,7 +6,8 @@ var path = require('path');
 var packageJson = require(path.resolve('./package.json'));
 var argv = require('yargs').argv;
 
-var OUTPUT_DIR = 'out' + Math.ceil(Math.random() * 9999);
+var SKIP_BUILD = !!argv['existing-output-dir']
```
Comment (ankurp) [inline bin/storybook_to_ghpages]: You can use `Boolean(argv['existing-output-dir'])` instead of `!!` to be more clear.

**Comment by phillymojo** [comment]
I was going to suggest this same exact functionality; is this something that can be merged?

**Comment by ndelangen** [comment]
Yes i can merge his if the merge-conflicts have been resolved, Do you think you could resolve them @travisbloom ?

**Comment by travisbloom** [comment]
hi all, sorry for the delay. Please let me know if there's anything else needed to get this merged :)

**Comment by ndelangen** [comment]
Thanks for this @travisbloom LGTM ✅

**Comment by ndelangen** [comment]
Published as `2.2.0`

**Pull request merged**